recipes/numdifftools/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "numdifftools" %}
+{% set version = "0.9.20" %}
+{% set sha256 = "443b43e43c970b4f37cd2a5a57f842192da89daa2d58105ca5f5f894b7d4fcd0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}-py2.py3-none-any.whl 
+  url: https://files.pythonhosted.org/packages/py2.py3/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}-py2.py3-none-any.whl 
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  # I build from the wheel so that we don't need pyscaffold. The version
+  # needed by the source distribution of numdifftools is not compatible with conda-forge
+  script: pip install {{ name }}-{{ version }}-py2.py3-none-any.whl --no-deps
+  noarch: python
+  
+requirements:
+
+  build:
+  
+    - python
+    - setuptools
+    - pip
+    - numpy >=1.9
```

Comment:
please see [here](https://conda-forge.org/docs/meta.html#building-against-numpy) for pinning against numpy.

Reply:
I am a bit confused. I thought that was relevant for compiling against numpy (for packages using the C/C++ API of numpy), while `numdifftools` is a pure-python package... am I wrong? why should it matter which numpy is used in the build stage? Since I did not use `numpy x.x`, I am not pinning against a specific numpy, am I?

Reply:
Is numpy a build dependency?

Reply:
Yes, but just because otherwise pip will try to download it (and the build will fail).